code/mob/new_player.dm

```diff
@@ -93,6 +93,8 @@ mob/new_player
 
 			else
 				spawned_in_keys += "[src.ckey]"
+				for (var/sound in global.dj_panel.preloaded_sounds)
+					src.client << load_resource(sound, -1)
```

Comment:
so if i preload like 20 songs and play them once with no intention of playing them ever again, any new clients would have to load those 20 songs?
can we add a way to delete preloaded songs or maybe make them single use somehow by default?

Reply:
good point, sounds now get removed from the preloaded list after being played

Reply:
i mean it might be useful if i load `buttfart.ogg` and want to play it a bunch of times idk

Reply:
there isn't much value to using this for short sounds since it's just more clicks (play preload -> select sound) for every play, and a 2 second sound isn't going to lag anyone anyways
also I don't trust people to remove preloaded sounds manually. realistically if 20 things get preloaded and people have to manually unload them, 20 things are staying preloaded